be/src/env/env_broker.cpp

```diff
@@ -454,10 +454,18 @@ Status EnvBroker::create_dir_if_missing(const std::string& dirname, bool* create
     return Status::NotSupported("BrokerEnv::create_dir_if_missing");
 }
 
+Status EnvBroker::create_dir_recursive(const std::string& dirname) {
+    return Status::NotSupported("BrokerEnv::create_dir_recursive");
```

Comment:
Better to replace `BrokerEnv` with `EnvBroker` in all status messages.

Reply:
Good finding.

Reply:
Fixed, PTAL